internal/db/sqltest/tests/hcp/billing/monthly_timezone.sql

```diff
@@ -0,0 +1,110 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: MPL-2.0
+
+begin;
+  select plan(18);
+
+  select lives_ok('truncate wh_session_connection_accumulating_fact, wh_session_accumulating_fact',
+                  'Truncate tables in preparation for testing');
+
+  -- validate the warehouse fact tables are empty
+  select is(count(*), 0::bigint, 'wh_session_connection_accumulating_fact is not empty') from wh_session_connection_accumulating_fact;
+  select is(count(*), 0::bigint, 'wh_session_accumulating_fact is not empty' ) from wh_session_accumulating_fact;
+
+  -- validate the view returns no rows
```

Comment:
I looks like some of these *do* return rows. Should this confirm that the resulting counts are `0` for the rows that do get returned?

Reply:
I will add tests for that.